cubi_tk/snappy/kickoff.py

```diff
@@ -12,46 +12,63 @@
 from cubi_tk.exceptions import ParseOutputException
 
 from . import common
+from .snappy_workflows import SnappyWorkflowManager
+
+# from .snappy_workflows import get_snappy_step_directories, get_workflow_step_dependencies
 
 
 def run(
     args, _parser: argparse.ArgumentParser, _subparser: argparse.ArgumentParser
 ) -> typing.Optional[int]:
     logger.info("Try to find SNAPPY pipeline directory...")
     try:
-        path = common.find_snappy_root_dir(args.path or os.getcwd(), common.DEPENDENCIES.keys())
+        path = common.find_snappy_root_dir(args.path or os.getcwd())
     except common.CouldNotFindPipelineRoot:
         return 1
 
-    # TODO: this assumes standard naming which is a limitation...
-    logger.info("Looking for pipeline directories (assuming standard naming)...")
+    logger.info("Looking for pipeline directories (needs to contain snappy config.yaml)...")
     logger.debug("Looking in %s", path)
-    step_set = {name for name in common.DEPENDENCIES if (path / name).exists()}
+
+    manager = SnappyWorkflowManager.from_snappy()
+
+    if manager is None:
+        return None
```

Comment:
I think it would also be okay to raise an error (with a nice message) here, since the `kickoff` function can rightfully depend on snappy installation. The warning given by the manager also conveys the necessray information though.